.golangci.yml

```diff
@@ -82,6 +82,7 @@ issues:
       linters:
         - dupl
         - funlen
+        - gochecknoglobals
```

Comment:
I generally prefer to exclude the logger warning rather than disable the globals linter.

Reply:
ok, this is only for test files so that's why I thought it was ok

Reply:
oh ok - I missed that this was for test files.